docker-compose.yml

```diff
@@ -46,7 +46,7 @@ services:
     depends_on: [postgres, redis, elasticsearch, mailcatcher, minio]
     build:
       context: ./server
-      dockerfile: Dockerfile.local
+      dockerfile: Dockerfile
```

Comment:
Can you revert this one line? This was intentional, I just forgot to push the Dockerfile.local and did that now 😅